Client/Networking/NetworkManager.swift

```diff
@@ -28,7 +28,8 @@ class NetworkManager {
     }
     
     func downloadResource(with url: URL, resourceType: NetworkResourceType,
-                          retryTimeout: TimeInterval? = 60) -> Deferred<CachedNetworkResource> {
+                          retryTimeout: TimeInterval? = 60,
+                          catchLastModifiedParam: Bool = false) -> Deferred<CachedNetworkResource> {
```

Comment:
Let's call this `cacheLastModifiedTimestamp` or `cacheLastModifiedDate`

Reply:
I renamed to `checkLastServerSideModification` since this is not a local file timestamp, open for other suggestions too

Reply:
That works for me and probably makes more sense.